variables.tf

```diff
@@ -369,95 +369,109 @@ variable "custom_security_group_ids" {
 }
 
 ##############################################################################
-# Log Analysis Agent Variables
+# Logs Agents variables
 ##############################################################################
 
-variable "log_analysis_enabled" {
+variable "logs_agent_enabled" {
   type        = bool
-  description = "Deploy IBM Cloud Logging agent"
+  description = "Whether to deploy the Logs agent."
   default     = true
 }
 
-variable "log_analysis_add_cluster_name" {
-  type        = bool
-  description = "If true, configure the log analysis agent to attach a tag containing the cluster name to all log messages."
-  default     = true
-}
-
-variable "log_analysis_secret_name" {
+variable "logs_agent_name" {
+  description = "The name of the Logs agent. The name is used in all Kubernetes and Helm resources in the cluster."
   type        = string
-  description = "The name of the secret which will store the ingestion key."
-  default     = "logdna-agent"
+  default     = "logs-agent"
   nullable    = false
 }
 
-variable "log_analysis_instance_region" {
+variable "logs_agent_namespace" {
   type        = string
-  description = "The IBM Log Analysis instance region. Used to construct the ingestion endpoint."
-  default     = null
+  description = "The namespace where the Logs agent is deployed. The default value is `ibm-observe`."
+  default     = "ibm-observe"
+  nullable    = false
 }
 
-variable "log_analysis_ingestion_key" {
+variable "logs_agent_iam_api_key" {
   type        = string
-  description = "Ingestion key for the Log Analysis agent to communicate with the instance."
+  description = "The IBM Cloud API key for the Logs agent to authenticate and communicate with the IBM Cloud Logs. It is required if `logs_agent_iam_mode` is set to `IAMAPIKey`."
```

Comment:
required if `logs_agent_enabled` is true and `logs_agent_iam_mode` is set to `IAMAPIKey`. Perhaps add validation for this

Reply:
<img width="1238" alt="Screenshot 2024-10-21 at 4 08 40 PM" src="https://github.com/user-attachments/assets/1dc941e0-53cc-4d0d-a230-757cf61e7634">